src/note.ts

```diff
@@ -13,15 +13,254 @@
 import { Vex } from './vex';
 import { Flow } from './tables';
 import { Tickable } from './tickable';
+import { Stroke } from './strokes';
+import { Stave } from './stave';
+import { BoundingBox } from './boundingbox';
+import { Voice } from './voice';
+import { TickContext } from './tickcontext';
+import { ModifierContext } from './modifiercontext';
+import { Modifier } from './modifier';
+import { RenderContext } from './types/common';
+import { Fraction } from './fraction';
+import { Beam } from './beam';
+
+export const GLYPH_PROPS_VALID_TYPES: Record<string, Record<string, string>> = {
```

Comment:
Does this need to be exported? I'd prefer to move this into `parseNoteStruct` below where it's used.

Reply:
It does not need to be exported it is a intermediate solution until `table.js` is migrated. This type is also defibed there.

Reply:
Yeah, I see. Is there a better way to do this? I'm not a fan of this duplication. Maybe we can have a separate file for this intermediate stuff and reference that?

Reply:
`common` with `TODO`?